_haikus/game_of_clouds.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Game of Clouds
+author: Rishab Menon
+---
+
+Iron Throne win who? <br>
+Google, Microsoft, AWS? <br>
+Amazon it is! <br>
```

Comment:
No love for DigitalOcean? ;)

Reply:
hahah..couldn't fit into it, or you know who is always the winner! ;)